ee/tabby-webserver/src/service/repository.rs

```diff
@@ -17,12 +17,16 @@ impl RepositoryService for DbConn {
         before: Option<String>,
         first: Option<usize>,
         last: Option<usize>,
-    ) -> Result<Vec<Repository>> {
+    ) -> Result<(Vec<Repository>, usize)> {
         let (limit, skip_id, backwards) = graphql_pagination_to_filter(after, before, first, last)?;
+        let repositories_count = self.count_rows("repositories").await?;
```

Comment:
does it make more sense to add count_repositories as a separate method of service?

Reply:
gotcha - let's hold this PR a bit, maybe after the #1340

Reply:
Sounds good to me